### DISCUSSION TITLE
What's the current state of this project?

### DISCUSSION THREAD

**Discussion by dword-design** [acss-io/atomizer#427] [General]
Hey folks, what's actually the current state of this project? Is it actively maintained?

**Comment by thierryk** [comment]
@dword-design I know you have PRs waiting.
You may want to *request* a review for those.

**Reply by dword-design** [reply]
Thanks for the hint 👍

**Reply by thierryk** [reply]
You're welcome. I'm glad to see it's moving in the right direction ("lentement mais sûrement").

**Comment by redonkulus** [comment]
Linking my reply from twitter: https://twitter.com/redonkulus/status/1382542722452492289?s=20
